src/config/dark-sites.config

```
@@ -614,6 +614,7 @@ redditinvestigator.com
 redeclipse.net
 reelgood.com
 refind.com
+regex101.com/
```

```suggestion
regex101.com
```
the `/` is not necessary.